pkg/cluster/k8sres.go

```diff
@@ -525,6 +527,52 @@ func (c *Cluster) generatePodTemplate(
 		)
 	}
 
+	monitoringContainerImage := c.OpConfig.MonitoringDockerImage
+	if monitoringDockerImage != nil && *monitoringDockerImage != "" {
+		monitoringContainerImage = *monitoringDockerImage
+	}
+	if monitoringContainerImage != "" {
+		podSpec.Containers = append(
+			podSpec.Containers,
+			v1.Container{
+				Name:            "monitoring-sidecar",
+				Image:           c.monitoringImage(),
```

Comment:
Looks like it is not defined anywhere.

Reply:
yea this was a sketch and I was trying to figure out how to get my environment set up... all fixed up and working now.